src/lime/tools/AIRHelper.hx

```diff
@@ -258,8 +291,8 @@ class AIRHelper
 				System.runCommand("", "killall", ["iPhone Simulator"], true, true);
 			}
 
-			System.runCommand(workingDirectory, project.defines.get("AIR_SDK") + "/bin/adt",
-				["-uninstallApp"].concat(args).concat(["-appid", project.meta.packageName]), true, true);
+//			System.runCommand(workingDirectory, project.defines.get("AIR_SDK") + "/bin/adt",
+//				["-uninstallApp"].concat(args).concat(["-appid", project.meta.packageName]), true, true);
```

Comment:
This will need to be cleaned up. Either delete or create a compiler flag to toggle it.